### ISSUE TITLE
Package not showing dependencies

### ISSUE THREAD

**Issue by jacekmackiewicz** [coverlet-coverage/coverlet#1439]
Package is not showing dependencies although having dependencies on example:
Microsoft.Extensions.DependencyInjection.Abstractions version 6.0.0 which is copied into project only when getting detailed publish information.
https://www.nuget.org/packages/coverlet.collector/3.2.0#dependencies-body-tab

**Comment by MarcoRossignoli** [comment]
Historically the collector was build without exposing any dependencies and we do a diff during the `dotnet publish` https://github.com/coverlet-coverage/coverlet/blob/master/src/coverlet.collector/build/netstandard1.0/coverlet.collector.targets#L12

are you experiencing loading issues?

**Comment by jacekmackiewicz** [comment]
Had a problem when all of the project packages use Microsoft.Extensions.DependencyInjection.Abstractions version 7.0.0 but coverlet-coverage copied over it's own version 6.0.0. It happened during dotnet publish and I couldn't track what's happening until I digged deeper into diagnostic messages. My app ended up throwing and error it couldn't find Microsoft.Extensions.DependencyInjection.Abstractions version 7.0.0.
Our solution was to exclude projects using coverlet-coverage at publish stage but I though it would be easier to find the dependecies while running dotnet list commands.
I noticed the diff during dotnet publish feels little bit sketchy, but maybe it's only me seeing it this way.
